netsim/modules/vxlan.py

```diff
@@ -146,7 +146,7 @@ def build_vtep_list(vlan: Box, node: str, nodes: typing.List[str], topology: Box
     ndata = topology.nodes[n]
     if not 'vlans' in ndata:                                        # No VLANs on remote node, skip it
       continue
-    vni_match = filter(lambda x: x.vni == vlan.vni,ndata.vlans.values())
+    vni_match = filter(lambda x: x.get('vni',0) == vlan.vni,ndata.vlans.values())
```

Comment:
Not sure if we allow VNI value zero, but None would definitely be a better option.

Reply:
This PR was carefully crafted considering all constraints: https://github.com/ipspace/netlab/blob/5b580c076b7f1077688f2475d0e1da278debeaad/netsim/modules/vlan.yml#L80

Reply:
...but I'll go with your suggestion